public/server/db/g8.json

```diff
@@ -0,0 +1,3 @@
+{
+  "definition": "Great"
```

Comment:
I believe the slang for great is gr8, and that is already included in the database.